module/data/item/templates/equippable-item.mjs

```diff
@@ -20,6 +20,25 @@ export default class EquippableItemTemplate extends SystemDataModel {
     };
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * Create attunement filter configuration.
+   * @returns {CompendiumBrowserFilterDefinitionEntry}
+   */
+  static get compendiumBrowserAttunementFilter() {
+    return {
+      label: "DND5E.Attunement",
+      type: "boolean",
+      createFilter: (filters, value, def) => {
+        if ( value === 0 ) return;
+        const filter = { k: "system.attunement", o: "in", v: ["required", 1] };
```

Comment:
Is this for legacy data?

Reply:
Yeah, because we are working with indexes we cannot guarantee that this data has been migrated so we need to be a bit more careful of what we check.